src/database.js

```diff
@@ -99,6 +126,22 @@ export class Database {
     return factory ? factory.serializer : undefined; 
   }
 
+  randomRecord(collectionName) {
+    return this.randomRecords(collectionName)[0];
+  }
+
+  randomRecords(collectionName, limit = 1) {
+    const all = this.all(collectionName);
+    const records = [];
+
+    while (limit) {
```

Comment:
If we use this we have to increment `limit` before the loop, which I think it's more confusing in the end... what do you think?